app/javascript/dashboard/App.vue

```diff
@@ -17,12 +19,21 @@ export default {
 
   components: {
     WootSnackbarBox,
+    AddAccountModal,
   },
 
   computed: {
     ...mapGetters({
       getAccount: 'accounts/getAccount',
+      currentUser: 'getCurrentUser',
     }),
+    showCreateAccountModal() {
```

Comment:
A suggestion on naming: It would better if it is something like `hasAccounts` or something similar.